rolemanager/module.py

```diff
@@ -0,0 +1,119 @@
+from typing import List
+
+import discord
+from discord.ext import commands
+
+from pie import utils, check, i18n
+from pie.utils.objects import ConfirmView, ScrollableEmbed
+
+_ = i18n.Translator("modules/fsi").translate
+
+
+# HELPER FUNCTIONS
```

Comment:
This comment should be lower (after __init__ func.)

Reply:
Corrected comment line.